vimeo-networking/src/main/java/com/vimeo/networking/model/error/ErrorCode.java

```diff
@@ -175,8 +175,6 @@
     UNABLE_TO_LOGIN_TOKEN_CAN_NOT_DECRYPT,
     @SerializedName("2314")
     UNABLE_TO_LOGIN_TOKEN_TOO_LONG,
-    @SerializedName("2315")
-    UNABLE_TO_LOGIN_PINCODE_EXPIRED,
```

Comment:
`INVALID_TOKEN` is the error that is sent from the API. Before this change, an expired pincode could technically result in two different `ErrorCode`s, `UNABLE_TO_LOGIN_PINCODE_EXPIRED` if we detect the invalidation at the client level, or `INVALID_TOKEN` if the invalidation is not detected by us, so this change brings the enum in line with the API's behavior. The numerical duplication means non determinism when it comes to serialization and deserialization, as when `UNABLE_TO_LOGIN_PINCODE_EXPIRED` was serialized to JSON and then deserialized, it would be deserialized as `UNABLE_TO_CREATE_USER_CAN_NOT_VALIDATE_TOKEN` instead.

Since this enum is meant to delineate error codes that can be returned by the API, putting in codes that can locally originate from our library means that the `ErrorCode` enum violates the single responsibility principle.